gstack-buffer/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "gstack-buffer"
```

Comment:
I'd really go with gear-stack-buffer, it doesn't look as good as short `g*` stuff

Reply:
I can do similar to gear-core-errors:
- folder: /stack-buffer/
- name in Cargo.toml: gear-stack-buffer

Reply:
that's ok